assignments/arrays.js

```diff
@@ -75,29 +75,50 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+console.log(`Car 33 is a *2011* *jeep* *wrangler*`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(inventory[inventory.length - 1].car_make);
+console.log(inventory[inventory.length - 1].car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
 let carModelsSorted = [];
 console.log();
 
+for( let i = 0; i < inventory.length; i++) {
+  carModels.push(inventory[i].car_model);
+}
+console.log(carModels);
+carModelsSorted = carModels.sort();
+console.log(carModelsSorted);
+
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
 console.log();
+for(let i = 0; i < inventory.length; i++) {
+  carYears.push(inventory[i].car_year);
+}
+// console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars = [];
 console.log();
 
+for(let i = 0; i < carYears.length; i++) {
+  oldCars.push(carYears[i]);
```

Comment:
you don't need to do this here. You are pushing all the car years into the oldCars array when you should only be pushing the cars with years less than 2000. 

You should be doing this on line 117 instead. If the car year is less than 2000, only then should you push. 

Also notice that you are using a greater than instead of a less than sign.

The last thing is that the question said you should log the length of the array. Not the entire array